templates/slurm.txt

```diff
@@ -277,17 +294,23 @@ Order = 10
         ParameterType = Cloud.MachineType
         DefaultValue = Standard_D8as_v4
 
+        [[[parameter HTCMachineType]]]
```

Comment:
Again, try to avoid diffs like this where the only change is moving a block - it makes it hard to differentiate between the meaningful changes and code cleanup.

(When we want to re-order or cleanup the file, that deserves a separate commit.   But this isn't a case that needs re-ordering anyway.)